config/locales/diaspora/en.yml

```diff
@@ -345,6 +345,10 @@ en:
     tutorials: "tutorials"
     tutorial: "tutorial"
     irc: "IRC"
+    closed_pod: "You can only join a pod that currently lists signups as ”Open”. (Remember, they all connect to the same network, you're not missing anything!)"
```

Comment:
I would word it this way: *"This pod registrations are closed. However, you can still join the diaspora\* network by registering on <a href="https://diasporafoundation.org/getting_started/sign_up">another pod</a>. Because all pods are interconnected, you will have access to the same content there."*.

I link to the official website tutorials page here, I think it should be the central point where we redirect new comers for more information. There they will find everything they need to register elsewhere, no need to directly link poduptime or the wiki.

Reply:
Minor grammatical correction:

> *This pod is currently closed to new registrations. However, you can still join the diaspora\* network by registering on another pod [https://diasporafoundation.org/getting_started/sign_up]. Because all pods are interconnected, you will have access to the same content there.*